src/Processing.js

```diff
@@ -4066,10 +4066,36 @@
     p.noLoop = function() {
       doLoop = false;
       loopStarted = false;
-      clearInterval(looping);
+      
+      looping = false;
       curSketch.onPause();
     };
 
+requestAnimationFrame= (function(){
+      return window.requestAnimationFrame ||
+             window.webkitRequestAnimationFrame ||
+             window.mozRequestAnimationFrame    ||
+             function( callback ){
+                var _h= function(){callback(Date.now());};
+                window.setTimeout(_h, curMsPerFrame);
+             };
+     })();
+
+    var _now, _then=0, _doDraw = function(t) {
```

Comment:
missing space before `=` in most of the lines in this block (somewhat of a nit, but the rest of the code uses spaces around `=`, so it's better to stick to that style)